packages/framework/src/DependencyInjection/ShopsysFrameworkExtension.php

```diff
@@ -10,10 +10,12 @@
 use Shopsys\FrameworkBundle\Twig\VarDumperExtension;
 use Symfony\Component\Config\FileLocator;
 use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\DependencyInjection\Extension\PrependExtensionInterface;
 use Symfony\Component\DependencyInjection\Loader\YamlFileLoader;
 use Symfony\Component\HttpKernel\DependencyInjection\Extension;
+use Symfony\Component\Yaml\Yaml;
 
-class ShopsysFrameworkExtension extends Extension
+class ShopsysFrameworkExtension extends Extension implements PrependExtensionInterface
```

Comment:
I thought it is exactly described in the commit message, because it says:
"... that is configurable from both application and framework bundle"

And that is exactly what this change does. By default, you can configure your bundles only from your project - in our case project-base. We would be unable to add new queues to framework without BC break. This extension of ShopsysFrameworkExtension enables us to add producers and consumers without breaking our BC promise.

Reply:
ok, however, please include some information about that in the docs - I really do not know how it works now - the configs are merged in the end? Or the config in `project-base` has higher priority and overwrites the one in `framework`?  would really like to be the one who reviews the docs :grin:

Reply:
I will